catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/BroadcastAndWaitBrickTest.java

```diff
@@ -61,33 +63,46 @@
 public class BroadcastAndWaitBrickTest {
 	private String defaultMessage = "defaultMessage";
 
-	private int broadcastAndWaitPosition = 1;
+	private int firstBroadcastAndWaitPosition = 1;
+	private int secondBroadcastAndWaitPosition = 2;
 
 	@Rule
 	public BaseActivityInstrumentationRule<SpriteActivity> baseActivityTestRule = new
 			BaseActivityInstrumentationRule<>(SpriteActivity.class, SpriteActivity.EXTRA_FRAGMENT_POSITION, SpriteActivity.FRAGMENT_SCRIPTS);
 
 	@Before
 	public void setUp() throws Exception {
-		createProject("BroadcastAndWaitBrickTest");
+		createProject(this.getClass().getName());
 		baseActivityTestRule.launchActivity();
 	}
 
+	@After
+	public void tearDown() throws Exception {
+		TestUtils.deleteProjects(this.getClass().getName());
+	}
+
 	@Category({Cat.AppUi.class, Level.Functional.class})
 	@Test
 	@Flaky
 	public void testBroadcastAndWaitBrick() {
 		String uselessMessage = "useless";
-		createNewMessageOnSpinner(R.id.brick_broadcast_spinner, broadcastAndWaitPosition, uselessMessage);
+		createNewMessageOnSpinner(R.id.brick_broadcast_spinner, firstBroadcastAndWaitPosition, uselessMessage);
```

Comment:
please create separate tests for issues like this. do not extend existing tests to additionally test some other stuff on the way.

Reply:
done